torchgeo/datasets/levircd.py

```diff
@@ -262,3 +184,204 @@ def get_rgb(img: Tensor) -> "np.typing.NDArray[np.uint8]":
+            "md5": "a638e71f480628652dea78d8544307e4",
+        },
+        "val": {
+            "url": "https://drive.google.com/file/d/1BqSt4ueO7XAyQ_84mUjswUSJt13ZBuzG",
+            "filename": "val.zip",
+            "md5": "f7b857978524f9aa8c3bf7f94e3047a4",
+        },
+        "test": {
+            "url": "https://drive.google.com/file/d/1jj3qJD_grJlgIhUWO09zibRGJe0R4Tn0",
+            "filename": "test.zip",
+            "md5": "07d5dd89e46f5c1359e2eca746989ed9",
+        },
+    }
+
+    def _load_files(self, root: str, split: str) -> list[dict[str, str]]:
+        """Return the paths of the files in the dataset.
+
+        Args:
+            root: root dir of dataset
+            split: subset of dataset, one of [train, test]
+
+        Returns:
+            list of dicts containing paths for each pair of image1, image2, mask
+        """
+        images1 = sorted(glob.glob(os.path.join(root, "A", f"{split}*.png")))
+        images2 = sorted(glob.glob(os.path.join(root, "B", f"{split}*.png")))
+        masks = sorted(glob.glob(os.path.join(root, "label", f"{split}*.png")))
+
+        files = []
+        for image1, image2, mask in zip(images1, images2, masks):
+            files.append(dict(image1=image1, image2=image2, mask=mask))
+        return files
+
+    def _check_integrity(self) -> bool:
+        """Checks the integrity of the dataset structure.
+
+        Returns:
+            True if the dataset directories and split files are found, else False
+        """
+        return all(
+            [
+                os.path.exists(os.path.join(self.root, directory))
+                for directory in self.directories
+            ]
+        )
+
+    def _download(self) -> None:
+        """Download the dataset and extract it."""
+        if self._check_integrity():
+            print("Files already downloaded and verified")
+            return
+
+        for split in self.splits:
+            download_and_extract_archive(
+                self.splits[split]["url"],
+                self.root,
+                filename=self.splits[split]["filename"],
+                md5=self.splits[split]["md5"] if self.checksum else None,
+            )
+
+
+class LEVIRCDPlus(LEVIRCDBase):
+    """LEVIR-CD+ dataset.
+
+    The `LEVIR-CD+ <https://github.com/S2Looking/Dataset>`__
+    dataset is a dataset for building change detection.
+
+    Dataset features:
+
+    * image pairs of 20 different urban regions across Texas between 2002-2020
+    * binary change masks representing building change
+    * three spectral bands - RGB
+    * 985 image pairs with 50 cm per pixel resolution (~1024x1024 px)
+
+    Dataset format:
+
+    * images are three-channel pngs
+    * masks are single-channel pngs where no change = 0, change = 255
+
+    Dataset classes:
+
+    1. no change
+    2. change
+
+    If you use this dataset in your research, please cite the following paper:
+
+    * https://arxiv.org/abs/2107.09244
+
+    .. versionchanged:: 0.6
```

Comment:
Not true

Reply:
Why would this not be true? We already had a LEVIRCDPlus class and this PR changed it. There are no breaking changes to it.

Reply:
Ah, I read this as versionadded the first time. Still, we should say _what_ changed. Or did the API not change?

Reply:
I can add that in another PR. I just made a base class so I could override a couple methods for loading the files since they have different directory setups. The LEVIRCDPlus itself hasn't changed though other than inheriting from the new base class

Reply:
In that case I would remove versionchanged, there are no significant API differences that users need to be aware of.